server/@types/approved-premises/index.d.ts

```diff
@@ -45,5 +46,12 @@ declare module 'approved-premises' {
       bedCount: number
       apAreaId: string
     }
+    Booking: {
+      id?: string
```

Comment:
As mentioned in the commit message this Id is nullable as it won't be there on the request but it will be on the response (assuming it's created on the server). 
I'm not sure if this is a good approach or if we should have two seperate types to more clearly express what's going on

Reply:
Yeah, I think having a `BookingDto` would be better. What I've done in my PR is using the `Omit` utility class, so for booking, it'd look something like this:

```
type BookingDto = Omit<Booking, 'id'>
```